src/models/LocalNewsModel.js

```diff
@@ -0,0 +1,36 @@
+// @flow
```

Comment:
I noticed that we use some properties and methods on local news and Tu News like title, message, date and id etc, so i am thinking to create parent class like NewsStream and extend it on local news and Tu News instead of repeating the code. What do you think?